### PR TITLE
SINKCORR and forward model doc updates

### LINKED ISSUE #68: Update documentation
Currently, the documentation for acstools contains no references to SINKCORR in the [calacs.e](https://acstools.readthedocs.io/en/stable/calacs_hstcal.html) section. In that same section, it also tells users that CALACS may be downloaded as part of the HSTCAL package in STSDAS. So we should update the docs to include links to astroconda and the SINKCORR step.

### PR DESCRIPTION
**Pull request by jryon**
Minor updates to the documentation. Includes documentation for SINKCORR and ACSCTEFORWARDMODEL. Closes #68

### CODE REVIEW

File: acstools/acscteforwardmodel.py [hunk 1]
```diff
@@ -8,6 +8,8 @@
 `ACS Data Handbook <http://www.stsci.edu/hst/acs/documents/handbooks/currentDHB/>`_
 for more details.
 
+For guidance on running the CTE forward model, see the Jupyter notebook `ACS CTE Forward Model Example <https://github.com/spacetelescope/acs-notebook/blob/master/acs_cte_forward_model/acs_cte_forward_model_example.ipynb>`_.
```
Comment (pllim) [inline acstools/acscteforwardmodel.py]: Nitpick, please break this long sentence into multiple lines, preferably to have line breaks every 80th char. :wink:

File: acstools/acscteforwardmodel.py [hunk 2]
```diff
@@ -8,6 +8,10 @@
 `ACS Data Handbook <http://www.stsci.edu/hst/acs/documents/handbooks/currentDHB/>`_
 for more details.
 
+For guidance on running the CTE forward model, see the Jupyter notebook `ACS \
```
Comment (pllim) [inline acstools/acscteforwardmodel.py]: No need for trailing slash
```suggestion
For guidance on running the CTE forward model, see the Jupyter notebook `ACS
```

File: acstools/acscteforwardmodel.py [hunk 3]
```diff
@@ -8,6 +8,10 @@
...
+CTE Forward Model Example <https://github.com/spacetelescope/acs-notebook/blob\
```
Comment (pllim) [inline acstools/acscteforwardmodel.py]:
```suggestion
CTE Forward Model Example <https://github.com/spacetelescope/acs-notebook/blob
```

File: acstools/acscteforwardmodel.py [hunk 4]
```diff
@@ -8,8 +8,8 @@
 `ACS Data Handbook <http://www.stsci.edu/hst/acs/documents/handbooks/currentDHB/>`_
 for more details.
 
-For guidance on running the CTE forward model, see the Jupyter notebook `ACS \
-CTE Forward Model Example <https://github.com/spacetelescope/acs-notebook/blob\
+For guidance on running the CTE forward model, see the Jupyter notebook `ACS
+CTE Forward Model Example <https://github.com/spacetelescope/acs-notebook/blob
```
Comment (pllim) [inline acstools/acscteforwardmodel.py]: Oh, ops. I am actually not sure if Sphinx will render URL properly if it is broken up... Did you try to build the doc and see how this is rendered? You might have to make the URL text all one line (against my earlier comment). That is this all in one line:
```
`ACS CTE Forward Model Example <url here>`_
```
Sorry for any inconvenience caused!

Comment (jryon) [reply acstools/acscteforwardmodel.py]: I did build the doc to see if it rendered correctly, and it looked fine to me. It's possible I didn't build it correctly, though, so I will make the link all one line.

**Comment by pllim** [comment]
:horse: :skull: 

Thank you!

**Pull request merged**